File: Mixter/Domain/Identity/UserIdentities/Events/UserRegistered.cs
using Mixter.Domain.Core;

namespace Mixter.Domain.Identity.UserIdentities.Events
{
    public struct UserRegistered : IDomainEvent
    {
        public UserId UserId { get; private set; }

        public UserRegistered(UserId userId)
            : this()
        {
            UserId = userId;
        }

        public object GetAggregateId()
        {
            return UserId;
        }
    }
}

File: app/Domain/Identity/UserRegistered.php
<?php

namespace App\Domain\Identity;

use App\Domain\IDomainEvent;

class UserRegistered implements IDomainEvent {
    /**
     * @var string
     */
    private $userId;

    public function __construct($userId) {
        $this->userId = $userId;
    }

    /**
     * @return string
     */
    public function getUserId()
    {
        return $this->userId;
    }
}